src/Packagist/WebBundle/Resources/views/forms.html.twig

```diff
@@ -6,16 +6,16 @@
 {% endblock textarea_widget %}
 
 {% block form_widget_simple %}
-{% filter spaceless %}
+{% apply spaceless %}
```

Comment:
To match what is done in Symfony core since years (for better performance and better result regarding the intended output), the form theme should use whitespace control rather than the `spaceless` filter (better performance is because whitespace controll is a compile-time feature while spaceless is a runtime one, and rendering a form renders lots of these blocks)

Reply:
Done.